scripts/pyspelling/spellchecker.sh

```diff
@@ -54,11 +56,7 @@ else
     return 0 2>/dev/null
 fi
 
-pyspelling_run="Running pyspelling spellchecker on docs"
-echo "$line_block $pyspelling_run $line_block"
-
-# Update pyspelling.yml with a new source path
-yq e ".matrix[0].sources = [\"$source_to_spellcheck|!docs/includes/**/*.md\"]" -i "$pyspelling_yaml"
+echo "$line_block Running pyspelling spellchecker on docs $line_block"
 
-echo $source_to_spellcheck
-pyspelling -c ./$pyspelling_yaml
+echo "$source_to_spellcheck"
+pyspelling -c ./scripts/pyspelling/pyspelling.yml -S "$source_to_spellcheck"'|!docs/includes/**/*.md'
```

Comment:
This is nice, thanks.

Just to note though that the **task** needs to be specified with the `-n` flag for `-S` to work - [Pyspelling CLI docs](https://facelessuser.github.io/pyspelling/#command-line-usage).
```suggestion
pyspelling \
     -c ./scripts/pyspelling/pyspelling.yml \
    -n "Markdown docs" \
    -S "$source_to_spellcheck"'|!docs/includes/**/*.md'
```

Reply:
I'm a bit confused about this -- I saw that in the docs as well, but it seemed to work fine as long as there was only one task in the YAML. Let me play around with this a bit and confirm.

Reply:
I can add a comment to that effect if you think it would help.

Reply:
Yeah, please do check. I only had a small window to review this yesterday, so I may have overlooked something.

From testing,  the markdown files being targeted were not changing unless unless I added a task flag + arg.